application.go

```diff
@@ -169,7 +169,7 @@ func (c *Client) CreateApplication(config *ApplicationConfig) (*Application, err
 }
 
 // UpdateApplication updates an application
-func (c *Client) UpdateApplication(id string, application *UpdateApplicationRequest) (*Application, error) {
+func (c *Client) UpdateApplication(id string, application *ApplicationConfig) (*Application, error) {
```

Comment:
Why is this changed ? It should remain `UpdateApplicationRequest`

Reply:
IIRC, there were some issues I faced with it in the CLI and hence changed it to `*ApplicationConfig`
Also, to be consistent with https://github.com/civo/civogo/blob/master/kubernetes.go#L248 and https://github.com/civo/civogo/blob/master/loadbalancer.go#L154 

I have also changed CLI to use the config. Is there a problem changing this?

Reply:
Yes, the update endpoint needs the fields present in UpdateApplicationRequest. You have to change it back and figure out the issue in CLI/API

Reply:
Okay, updated this PR. PTAL.